integration_test/list_test.go

```diff
@@ -42,3 +47,260 @@ func TestKrewList(t *testing.T) {
 	}
 	// TODO(ahmetb): install multiple plugins and see if the output is sorted
 }
+
+func TestKrewListJSON(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.Krew("install",
+		"--manifest", filepath.Join("testdata", "foo.yaml"),
+		"--archive", filepath.Join("testdata", "foo.tar.gz")).
+		RunOrFail()
+
+	expected := []byte("[\n    {\n        \"Name\": \"foo\",\n        \"Version\": \"v0.1.0\"\n    }\n]\n")
```

Comment:
why do you compare json output?
maybe just parse and cmp lists?

Reply:
I had no idea if it was wanted or not, so better to throw it in than not. Plus, it helped inform me.